composer.json

```diff
@@ -7,6 +7,9 @@
     "guzzlehttp/psr7": "^1.6",
     "psr/http-message": "^1.0"
   },
+  "suggest": {
+    "google/cloud-storage": "Google Cloud Storage client library for storing and persisting files"
```

Comment:
Suggestion: I think this message should include a note about auto-registering the stream wrapper.

Reply:
good idea